### DISCUSSION TITLE
[Projects Beta] Input Japanese is broken on single select field

### DISCUSSION THREAD

**Discussion by toshihr** [community/community#6905] [Projects and Issues]
There is two problems for adding Japanese text on the single select field.
1. The first enter key pressing should be treated as a fixing translate by IME, but the key press is recognized as an adding new option. The second enter key pressing should be treated as the adding new option.
2. Also the value is copied to the new option

I think these two behaviors are bugs.
I'm very happy if these are fixed.

![74ef11d3791ece7f6f7877d8d2d6f6a2](https://user-images.githubusercontent.com/6064065/139251573-4129711f-e415-4b84-9e9c-02edc44484bf.gif)

**Comment by mesqueeb** [comment]
duplicate of https://github.com/github/feedback/discussions/6358
